packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -67,7 +61,7 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
     onFetchAvatarPersonaData,
     onRenderMessage,
     onRenderTypingIndicator,
-    onFetchParticipantMenuItems,
+    // onFetchParticipantMenuItems,  // Removed for GA release
```

Comment:
are we still keeping this comment?

Reply:
Yeah, I am trying to keep the diff small where I had to make intrusive changes / leave bread crumbs.